public/js/views/management.jsx

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
```

Comment:
what is this file used for? It seems like ``management/app.jsx`` replaced it, no?